app/views/games/show.html.slim

```diff
@@ -24,4 +24,4 @@
 
 .row.buttons
   .col-xs-12.text-center
-    = link_to 'Forfeit Game', forfeit_game_path(@game), class: 'btn btn-danger btn-lg', method: 'PUT' if @game.players.include?(current_user) && @game.winner.nil?
+    = link_to 'Forfeit Game', forfeit_game_path(@game), class: 'btn btn-danger btn-lg', method: 'PUT' if @game.winner.nil? && @game.full && @game.includes_player?(current_user)
```

Comment:
:+1: 

one idea to clean it up a tiny bit.

I'd move this logic to the `games_helper`

```
def current_user_can_forfeit?(game)
  return false unless current_user
  ## more code here...
end
```